lib/lotus/model/configuration.rb

```diff
@@ -137,6 +144,101 @@ def mapping(path=nil, &blk)
         @mapper_config = Lotus::Model::Config::Mapper.new(path, &blk)
       end
 
+      # Set the logger
+      #
+      # @param logger [Logger] the logger instance
+      #
+      # @example Set custom logger and retrieve the set logger
+      #   require 'lotus/model'
+      #
+      #   Lotus::Model.configure do
+      #     logging ::Logger.new(STDOUT)
+      #   end
+      #
+      #   Lotus::Model.configuration.logging
+      #     # => #<Logger:0x007ffc45227920>
+      #
+      #   Lotus::Model.configuration.logger
+      #     # => #<Logger:0x007ffc45227920>
+      #
+      # @since 0.3.0
+      # @see Lotus::Model.configure
+      def logging(logger = nil)
+        if logger.nil?
+          @logger
+        else
+          @logger ||= logger
+        end
+      end
```

Comment:
what is the difference bw logging and logger?, according to the sample, it's seem to be the same thing